src/System.Management.Automation/engine/debugger/debugger.cs

```diff
@@ -1553,16 +1565,21 @@ internal void OnSequencePointHit(FunctionContext functionContext)
             {
                 if (functionContext._breakPoints[functionContext._currentSequencePointIndex])
                 {
-                    var breakpoints = (from breakpoint in functionContext._boundBreakpoints
-                                       where
-                                           breakpoint.SequencePointIndex == functionContext._currentSequencePointIndex &&
-                                           breakpoint.Enabled
-                                       select breakpoint).ToList<Breakpoint>();
-
-                    breakpoints = TriggerBreakpoints(breakpoints);
-                    if (breakpoints.Count > 0)
+                    if (functionContext._boundBreakpoints.TryGetValue(functionContext._currentSequencePointIndex, out var bps))
                     {
-                        StopOnSequencePoint(functionContext, breakpoints);
+                        var breakpoints = (from breakpoint in bps
+                                        where
+                                            breakpoint.Enabled
+                                        select breakpoint).ToList<Breakpoint>();
```

Comment:
```suggestion
                        var enabledBreakpoints = new List<Breakpoint>();
                        foreach (Breakpoint breakpoint in bps)
                        {
                            if (breakpoint.IsEnabled)
                            {
                                enabledBreakpoints.Add(breakpoint);
                            }
                        }
```

Reply:
Thanks, I need to change names, and IsEnabled to Enabled. Will fix in code.